doc/designs/prci_checker.md

```diff
@@ -0,0 +1,149 @@
+# FreeIPA Pull Request CI (PR-CI) checker tool
+
+[PR-CI checker](https://github.com/freeipa/freeipa/blob/master/ipatests/prci_definitions/prci_checker.py) is a tool which checks that FreeIPA PR-CI definition files follow a pre-defined convention such as having the correct naming format, requirements and arguments.
+
+Usual usecase is to check the format of newly added jobs in the PR-CI definitions
+
```

Comment:
I think it would be clearer to read: The purpose is to check the format of newly added or modified jobs in the PR-CI definitions.

Reply:
Changed

Reply:
You can also add that the tool is intended for developers, not for users (it is not shipped with freeipa).

Reply:
added